Source/Fuse.Controls.Panels/SingleViewHost.uno

```diff
@@ -52,6 +52,11 @@ namespace Fuse.Controls
 
 			void NextFrame()
 			{
+				UpdateManager.PerformNextFrame(NextNextFrame);
```

Comment:
It'd be cleaner to modify PerformNextFrame to allowing skipping an extra frame -- it already has the ability to wait an arbitrary number of frames internally (`deferFrame`)

I think a comment here would also be helpful as to why waiting two frames is necessary.